packages/components/src/DateTimePickers/DateTime/Validation/Error.component.test.js

```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import { shallow } from 'enzyme';
+import { mount } from 'enzyme';
```

Comment:
Please restore this file, changing just the `Error.WrappedComponent` to `Error`.

The mount is not relevant here and it turn the snapshot to something that is not what we want (mount wrappers should be snapshot with enzyme-to-json, but here just a shallow is sufficient).